ssv/runner.go

```diff
@@ -48,6 +48,7 @@ type BaseRunner struct {
 	BeaconRoleType types.BeaconRole
 
 	// highestDecidedSlot holds the highest decided duty slot and gets updated after each decided is reached
+	// used to validate pre-consensus justification which can not have any lower duty slot than highestDecidedSlot
 	highestDecidedSlot spec.Slot
```

Comment:
now that the controller's height is also the highest decided slot are we sure we really need this?

Reply:
we can remove it and use the controller's height instead (please add tests)